packages/react-component-library/src/enhancers/index.tsx

```diff
@@ -0,0 +1,13 @@
+import * as Components from '../components'
+import withFormik from './withFormik'
+
+const Formik: object = {
+  // Enhanced components
+  // ...
+}
+
+Object.keys(Components).forEach(key => {
```

Comment:
You are enhancing every component, not just form ones?

I'd have just picked ones that can be enhanced

Reply:
Haha yes good point! I'll change!